app/src/main/java/com/concordium/wallet/core/security/BiometricPromptCallback.kt

```diff
@@ -5,12 +5,16 @@ import com.concordium.wallet.util.Log
 import javax.crypto.Cipher
 
 open class BiometricPromptCallback : BiometricPrompt.AuthenticationCallback() {
+    companion object{
+        const val ERROR_NEGATIVE_BUTTON = 13
+        const val ERROR_USER_CANCELED = 10
+    }
     override fun onAuthenticationError(errorCode: Int, errString: CharSequence) {
         super.onAuthenticationError(errorCode, errString)
         Log.d("Biometrics auth error (code: $errorCode): $errString")
-        if (errorCode == BiometricPrompt.ERROR_NEGATIVE_BUTTON) {
```

Comment:
Can you please elaborate on why we should replace library framework-defined error codes, with our own constants?

Reply:
With the old code I got this error: BiometricConstants.ERROR_NEGATIVE_BUTTON can only be accessed from within the same library (androidx.biometric:biometric).
Updating the project now flags this as an error. The app can be build with this error but just in case I've made the change.

Reply:
You mean that this was a warning/lint?

We should have probably jsut suppressed that instead of changing the code, but OK.